### ISSUE TITLE
Feature Request: Quick Copy, Paste with Indention & Expand Selection

### ISSUE THREAD

**Issue by chrisgrieser** [timhor/obsidian-editor-shortcuts#1]
Hi, thanks for this useful plugins! I like the idea, it's simple but still useful. I use Sublime and I'd love to see the following commands added to Obsidian via this plugin as well.
- Expand Selection to next match (Sublime Command:`find_under_expand`)
- Paste & Indent (Sublime: `paste_and_indent`)
- Expand to Brackets (Sublime: `expand_selection` to `brackets`)
- Quick Copy & Quick Cut: without selection, using copy or cut will instead copy/cut the whole line (Sublime: `copy_with_empty_selection`, found in the preference section, not the keybindings section)

**Comment by timhor** [comment]
Thanks for the feedback and suggestions! I'll address each one individually:

- Expand selection to next match: I can implement expanding the selection from a cursor to the word it's in – is that what you're after with this one? The multi-cursor aspect is out of scope at this stage as I haven't looked into how it works in Obsidian yet.
- Paste and indent: can you provide an example of your use case, since this feature seems more applicable to writing code?
- Expand to brackets: sure, I can look into this for the next version.
- Quick copy/cut: these already seem to work out of the box for me, is that not the case for you?

**Comment by timhor** [comment]
> Too bad, it's mostly the multiple selections of a term. 😕

Fair enough; I'd like to explore multiple selections at some point for sure, but no guarantees as I don't know what the API looks like, or if one even exists at the moment 😅  

> when copying content from one list to another one that has a different indention level, you have the issue that indention is wrong, I believe.

Ah I see, so it usually happens in nested lists. I'll keep it in mind for now, but from my experience, there are a number of edge cases to consider with nesting and paste (different users have different expectations) so it might not be as simple as it sounds.

**Comment by timhor** [comment]
Expand selection to next match is now available in https://github.com/timhor/obsidian-editor-shortcuts/releases/tag/1.8.0 under 'Select word or next occurrence of selection'!
